index.html

```diff
@@ -28,10 +27,9 @@
         <div class= "content">
```

Comment:
- [ ] Please use a `header` tag for the section that contains the logo and the hamburger button. This tag is meant to help with the structure of the HTML. Please use header that to wrap inside the logo and the hamburger button. 👍 🥇  
Remember that you must use one of the Figma templates.